src/main/java/io/github/moulberry/notenoughupdates/ItemPriceInformation.java

```diff
@@ -77,7 +80,9 @@ public static boolean addToTooltip(List<String> tooltip, String internalname, It
                             }
                             int bazaarSellPrice = (int) bazaarInfo.get("avg_sell").getAsFloat() * stackMultiplier;
                             tooltip.add(EnumChatFormatting.YELLOW.toString() + EnumChatFormatting.BOLD + "Bazaar Sell: " +
-                                    EnumChatFormatting.GOLD + EnumChatFormatting.BOLD + format.format(bazaarSellPrice) + " coins");
+                                    EnumChatFormatting.GOLD + EnumChatFormatting.BOLD +
```

Comment:
make this a method or something, it shouldn't be repeated this many times

Reply:
idk man i made it "more clean"